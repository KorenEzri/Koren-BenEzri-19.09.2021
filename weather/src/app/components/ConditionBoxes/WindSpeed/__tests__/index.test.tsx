import * as React from 'react';
import { render } from '@testing-library/react';

import { WindSpeed } from '..';


describe('<WindSpeed  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<WindSpeed />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
