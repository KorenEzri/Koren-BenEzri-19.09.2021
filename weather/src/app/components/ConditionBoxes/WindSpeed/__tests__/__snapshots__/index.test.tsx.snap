// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WindSpeed  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 img {
  width: 140px;
  height: 140px;
  margin: auto;
}

.c0 p {
  margin-top: -6px;
  margin-left: 3px;
  font-weight: bold;
  -webkit-letter-spacing: 1.5px;
  -moz-letter-spacing: 1.5px;
  -ms-letter-spacing: 1.5px;
  letter-spacing: 1.5px;
}

<div
  class="c0"
>
  <img
    alt="wind speed"
    src="wind-speed.png"
  />
  <p />
</div>
`;
